.github/workflows/upload-tutorials-stats.yml

```
@@ -0,0 +1,40 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Upload tutorials stat
+
+# Controls when the workflow will run
+on:
+  pull_request: #DO NOT MERGE
```

Nit: Let's do it on pull, when files in question are changed
```suggestion
  pull_request: 
    paths:
      - .github/workflows/upload-tutorials-stats.yml
      - .github/scripts/get_tutorials_stats.py
```